src/Microsoft.AspNetCore.Razor.Runtime/TagHelpers/TagHelperContent.cs

```diff
@@ -19,14 +19,9 @@ public abstract class TagHelperContent : IHtmlContentBuilder
         public abstract bool IsModified { get; }
 
         /// <summary>
-        /// Gets a value indicating whether the content is empty.
+        /// Gets a value indicating whether the content is empty or whitespace.
         /// </summary>
-        public abstract bool IsEmpty { get; }
-
-        /// <summary>
-        /// Gets a value indicating whether the content is whitespace.
-        /// </summary>
-        public abstract bool IsWhiteSpace { get; }
+        public abstract bool IsEmptyOrWhiteSpace { get; }
```

Comment:
We had a long discussion about what to expose on `TagHelperContent` when designing this API. In particular, the original decisions were _not_ just about what we needed in our tag helpers. So what is different now that drives removing the `IsEmpty` / `IsWhiteSpace` distinction? It's got to be something more than just avoiding fixing up one more property in this PR.

@Eilon? @DamianEdwards?